src/applications/ivc-champva/10-7959a/chapters/beneficiaryInformation.js

```diff
@@ -131,13 +135,13 @@ export const applicantPhoneSchema = {
   uiSchema: {
     ...titleUI(
       ({ formData }) =>
-        `${nameWording(formData, true, true, true)} contact information`,
+        `${nameWording(formData, true, true, true)} phone number`,
       ({ formData }) =>
         `We’ll use this information to contact ${
           formData?.certifierRole === 'applicant'
             ? 'you'
-            : nameWording(formData, true, true, true)
-        } if we have more questions.`,
+            : nameWording(formData, false, true, true)
+        } if we have any questions.`,
```

Comment:
Second boolean flag set to false to remove unnecessary capitalization of "your" when `certifierRole === 'applicant'`